src/langs/en.json

```
@@ -46,5 +46,12 @@
   "ENROLL_BUTTON": "Enroll",
   "ENROLL_DESCRIPTION": "Click on the button below to enroll and access this item.",
   "ITEM_LOGIN_HELPER_SIGN_OUT": "You're logged in as {{email}}",
-  "HOME_EMPTY": "You have no elements."
+  "HOME_EMPTY": "You have no elements.",
+  "AUTO_LOGIN_NO_ITEM_LOGIN_ERROR": "This item does not support auto-login",
+  "AUTO_LOGIN_ALREADY_LOGGED_IN": "You are already logged in with another account",
+  "AUTO_LOGIN_SIGN_OUT_AND_BACK_IN": "Sign Out and Log Back In",
```

```suggestion
  "AUTO_LOGIN_SIGN_OUT_AND_BACK_IN": "Log Out and Log Back In",
```
Log back in with auto login? Add this precision?

---

We are not making any mention of auto-login explicitly in this page, so I am not sure...